cli/unipipe/blueprints/terraform-runner-hello-world-service.ts

```diff
@@ -0,0 +1,156 @@
+      longDescription: "This simple service renders recipes in your git repository. It uses a terraform script that is executed by terraform-runner. Have a look in the terraform/ folder of your git repository to find out more."
+      providerDisplayName: "Pizza Inc."
+      documentationUrl: "https://docs.microsoft.com/en-us/azure/devops/?view=azure-devops"
+      supportUrl: "https://azure.microsoft.com/en-us/services/devops/"
+      tenantAware: true
+    tags:
+      - "pizza"
+    plans:
+      - id: "pizzab55-75be-4ec4-ad74-b62a07dd1650"
+        name: "Standard"
+        description: "A standard sized pizza."
+        free: false
+        metadata:
+          bullets:
+            - your choice of topping
+          costs:
+            - amount:
+                eur: 0.99
+              unit: MONTHLY
+            - amount:
+                eur: 1.00
+              unit: SETUP FEE
+          displayName: Standard (28 cm)
+        schemas:
+          service_instance:
+            create:
+              parameters:
+                $schema: http://json-schema.org/draft-04/schema#
+                type: object
+                properties:
+
+                  topping:
+                    title: "Topping"
+                    description: "Choose something you like."
+                    type: "string"
+                    widget: select
+                    enum:
+                      - "Funghi"
+                      - "Salami"
+`;
+}
+
+export const helloWorldMainTf = `terraform {
+  required_providers {
+    local = {
+      source = "hashicorp/local"
+      version = "~>2.2.3"
+    }
+  }
+}
+
+resource "local_file" "pizza_receipe" {
+  content  = <<EOT
+Preheat the oven to 200 degrees Celsius
+Form the pizza dough and place it on a peel dusted with a little flour. 
+Add sauce, cheese and \${var.topping}.
+Slip the pizza into the oven
+After 10 minutes of baking, check the pizza.
+EOT
```

Comment:
d: I am wondering whether we could also show usage of e.g. project_id and customer_id here. That could e.g. be the part of "Deliver pizza to \${var.customer_id}.\${var.project_id}"